src/main/register-injectables.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import type { DiContainer } from "@ogre-tools/injectable";
+import { autoRegister } from "@ogre-tools/injectable-extension-for-auto-registration";
+import { registerMobX } from "@ogre-tools/injectable-extension-for-mobx";
+import { runInAction } from "mobx";
+import { Environments, setLegacyGlobalDiForExtensionApi } from "../extensions/as-legacy-globals-for-extension-api/legacy-global-di-for-extension-api";
+
+export function registerInjectables(di: DiContainer) {
+  setLegacyGlobalDiForExtensionApi(di, Environments.main);
```

Comment:
Should there also be a counterpart for this in renderer?

Reply:
Probably but it is going to be a bit tricky because main process spawns renderer (== renderer gets initialized in a separate process). Basically to make it happen library should not bundle renderer at all, the importing application would need to build custom version of a renderer bundle and give that as source to app window.

Reply:
Already fixed.